docs/commands/command-line-interface.md

```diff
@@ -9,9 +9,15 @@ Get account number for the `<key>`
 ### --account_key --account=`<account>`
 Get the public key for `<account>`
 
-### --clear_send_ids   
+### --clear_send_ids
 Remove all send IDs from the database (dangerous: not intended for production use)
 
+### --compare_rep_weights
+_version 21.0+_  
+Displays a summarized comparison between the hardcoded bootstrap weights and representative weights from the ledger. Full comparison is output to logs. Optional [`--data_path`](#-data_pathpath).
+
+Amounts are displayed in $Mnano = 10^{30} raw$. Differences in total weight are due to unreceived (pending) blocks. Mismatched samples are obtained as the absolute difference between hardcoded and ledger weights. From the samples, a distribution $N(\mu, \sigma)$ is obtained. Values above $\mu + \sigma$ are displayed as outliers. Representatives found in the ledger but not hardcoded, with a large voting weight, are displayed as newcomers.
```

Comment:
Thoughts on a list-based formatting to more individually callout the different parameters? Something like this helps me line things up with the output a little better, but also maintains a bit of a flow through the different, related metrics:

```
* Differences between total weights (`hardcoded weight` and `ledger weight`) are due to unreceived (pending) blocks
* `mismatched`:
    * `samples` - the count of hardcoded reps used in evaluation mismatches
    * `total` - sum of the absolute difference between individual rep samples from hardcoded and ledger weights
    * `mean` - average (mean) of absolute differences between individual rep samples from hardcoded and ledger weights
    * `sigma` - from the samples, a distribution $N(\mu, \sigma)$ is obtained
* `outliers` - values above $\mu + \sigma$, for potential inspection
* `newcomers` - representatives with large voting weight found in the ledger but not hardcoded, for potential inspection
```

Reply:
Absolutely, good call. Doesn't fit with the other commands which are single line explanations but better have more than less 👍

Reply:
I modified your example a bit, what do you think?